packages/st2api/debian/st2api.init

```diff
@@ -18,8 +18,8 @@
 PATH=/sbin:/usr/sbin:/bin:/usr/bin
 DESC="st2api"
 NAME=st2api
-DAEMON=/usr/share/python/st2api/bin/$NAME
-DAEMON_ARGS="--config-file /etc/st2/st2.conf"
+DAEMON=/usr/share/python/st2api/bin/gunicorn_pecan
+DAEMON_ARGS="/usr/share/python/st2api/lib/python2.7/site-packages/st2api/gunicorn_config.py -k eventlet -b 127.0.0.1:9101 --threads 10 --workers 1"
```

Comment:
gunicorn does not allow you to pass arguments the same way you're passing them to the script. You can although redefine config pach using env variable https://github.com/StackStorm/st2/blob/master/st2api/st2api/gunicorn_config.py#L35-L36. I don't see it as an immediate concern, the amount of users who need to change the config is extremely low, but you believe it is a problem, feel free to pass the variable.